website/files/models/github.py

```diff
@@ -14,3 +14,7 @@ class GithubFolder(GithubFileNode, Folder):
 
 class GithubFile(GithubFileNode, File):
     version_identifier = 'ref'
+
+    def touch(self, auth_header, revision=None, ref=None, branch=None, **kwargs):
+        revision = revision or ref or branch
```

Comment:
That's about what I expected- thanks for the clear and helpful answer, @chrisseto . I'll move these tickets forward accordingly.